doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -11,12 +11,12 @@ Input files
 =============
 
 There are three types of files needed to execute a run: static datasets (*fix* files containing climatological
-information), files that depend on grid resolution and initial conditions, and model configuration files (such as namelists).
+information), files that depend on grid resolution, initial and boundary conditions, and model configuration files (such as namelists).
```

Comment:
Missing a comma before final conjunction.

> "resolution, initial, and boundary conditions"

https://www.thesaurus.com/e/grammar/s/serial-comma/#do-you-call-it-the-serial-comma-or-the-oxford-comma